### PR TITLE
Revert "feature #33507 [WebProfiler] Deprecated intercept_redirects in 4.4 (dorumd)"

### LINKED ISSUE #33467: [WebProfiler] Deprecate `intercept_redirects` in 4.4, remove it in 5.0
The `intercept_redirects` option is not needed anymore as the toolbar for the redirected resource now contains a link to the redirect response profile. For this reason, this option is set to `false` by default.

I think that this feature is not needed anymore and should be deprecated in 4.4 so that we can remove it in 5.0.

### PR DESCRIPTION
**Pull request by lyrixx**
This reverts commit 21a05de9bf196fe98d567c7bf0da14ce63922866, reversing
changes made to 24faadc905c6bc7096a284ac5ab3240cdfcff011.

| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? |
| Tickets       | Fix #33467
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php [hunk 1]
```diff
@@ -125,70 +117,13 @@ public function testToolbarConfig(bool $toolbarEnabled, bool $listenerInjected,
...
-            [
-                'toolbarEnabled' => true,
-                'interceptRedirects' => true,
-                'listenerInjected' => true,
-                'listenerEnabled' => true,
-            ],
+            [false, false, false, false],
+            [true,  false, true,  true],
+            [false, true,  true,  false],
+            [true,  true,  true,  true],
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php]: Can we at least keep the "good parts" like this?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php (hunk 1 above)
Comment (lyrixx) [reply src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php]: Sure! I'll do that ASAP

Comment (fabpot) [reply src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php]: friendly ping @lyrixx

Comment (lyrixx) [reply src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php]: Wep, Sorry I was AFK.

I re-reverted everything in the test folder

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**